src/pe.rs

```diff
@@ -152,6 +152,13 @@ where
         // characteristics flags, but these are obsolete.
         true
     }
+
+    #[inline]
+    fn has_debug_symbols(&self) -> bool {
+        // TODO: look at what the mingw toolchain does with DWARF-in-PE, and also
```

Comment:
File a follow-up investigation issue and link it here?

Reply:
Filed https://github.com/gimli-rs/object/issues/40